src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1790,7 +1788,11 @@ namespace UnitarySystems {
         // Delete next 2 lines and uncomment 2 lines inside next if (HeatPump) statement to allow non-heat pump systems to operate at different flow
         // rates (might require additional change to if block logic).
         EqSizing.CoolingAirVolFlow = max(EqSizing.CoolingAirVolFlow, EqSizing.HeatingAirVolFlow);
-        EqSizing.HeatingAirVolFlow = EqSizing.CoolingAirVolFlow;
+        if (this->m_HeatingCoilType_Num == DataHVACGlobals::Coil_HeatingWater) {
+            EqSizing.HeatingAirVolFlow = EqSizing.HeatingAirVolFlow;
+        } else {
+            EqSizing.HeatingAirVolFlow = EqSizing.CoolingAirVolFlow;
+        }
```

Comment:
Also operate the system at proper flow rate.

Reply:
So, the heating/cooling air flow at this point has already been sized to match either the main branch or oa system airflow, depending on the location of the unitary system?

Reply:
Yes, coil/system location is known. Just as the coil on branch flows are known during sizing.